### ISSUE TITLE
mysql: support for optimizer hints

### ISSUE THREAD

**Issue by mickeyreiss** [go-jet/jet#162]
**Is your feature request related to a problem? Please describe.**

I would like to generate the following SQL query:

```sql
SELECT /*+ MAX_EXECUTION_TIME(1000) */ * FROM orders;
```

This syntax is defined in the MySQL documentation [here](https://dev.mysql.com/doc/refman/5.7/en/optimizer-hints.html#optimizer-hints-execution-time).

**Describe the solution you'd like**
```go
SELECT(Orders.AllColumns).OptimizerHint("MAX_EXECUTION_TIME(100)").FROM(Orders)
```

**Comment by go-jet** [comment]
Yeah, it makes sense.

**Comment by go-jet** [comment]
MySQL optimizer hints included in v2.9.0 release.
